source/updateCheck.py

```diff
@@ -19,10 +19,11 @@
 )
```

Comment:
I'm uncomfortable by the rearrangement of imports in this file. There is a particular and fragile order here, and these changes largely do not seem relevant. Please revert them.